webcompat/static/js/lib/bugform.js

```diff
@@ -77,9 +77,14 @@ function BugForm() {
     );
     this.problemType.on("change", _.bind(this.checkProblemTypeValidity, this));
     this.uploadField.on("change", _.bind(this.checkImageTypeValidity, this));
-    this.osField
-      .add(this.browserField)
-      .on("blur input", _.bind(this.checkOptionalNonEmpty, this));
+    this.osField.on(
+      "blur",
+      _.bind(this.checkOptionalNonEmpty, this, ["osField"])
+    );
+    this.browserField.on(
+      "blur",
+      _.bind(this.checkOptionalNonEmpty, this, ["browserField"])
```

Comment:
Any reason to not just pass in the reference to the DOM elements directory here?

Reply:
I'm not sure I understand the wording. 🤔

Reply:
Let me rephrase:

You're changing `checkOptionalNonEmpty` to accept an argument `types`, which is an array of keys. For each of those keys, you grab a reference to the corresponding DOM input element via `input = this[key];`.

Why not just pass in `this.browserField` directly and skip passing the array of strings that ultimately just returns `this.browserField`?

`_.bind(this.checkOptionalNonEmpty, this, [this.browserField])`, etc.

Reply:
This is actually exactly what I did today. Small fix and will update the PR :)